app/assets/stylesheets/theme/_global.scss

```diff
@@ -5,4 +5,12 @@
   border-radius: $border-radius-base;
   border: 1px solid $gray-lighter;
   margin-bottom: 2em;
+  -webkit-box-shadow: 6px 6px 19px -2px rgba(0,0,0,0.75);
+  -moz-box-shadow: 6px 6px 19px -2px rgba(0,0,0,0.75);
+  box-shadow: 6px 6px 19px -2px rgba(0,0,0,0.75);
+}
+
+body {
+  background-image: image-url('bg-texture.png');
```

Comment:
If you change the BG image you're going to want to choose a BG color which comes "as close as possible" to the image, so the load doesn't look so shitty.